gubernator/kubelet_parser.py

```diff
@@ -23,28 +23,46 @@
 
 import regex
 
-def parse(lines, error_re, hilight_words, filters):
+def parse(lines, hilight_words, filters, objref_dict):
     """
-    Given filters returns indeces of wanted lines from the kubelet log
+    Given filters returns indeces of wanted lines from log
 
     Args:
-        lines: array of kubelet log lines
-        error_re: regular expression of the failed pod name
+        lines: array of log lines
         hilight_words: array of words that need to be bolded
         filters: dictionary of which filters to apply
```

Comment:
Addressed in new commit. It's a dictionary between possible filters and what string they would bold in the logs if the filter was on. Right now it only has information lifted from a line with an ObjectReference mention in the kubelet log, but the variable name should be changed as more filters are added.